package.json

```diff
@@ -68,8 +68,10 @@
     "jsonlint": "^1.6.2",
     "lodash.template": "^4.4.0",
     "minifyify": "^7.0.1",
+    "mock-geolocation": "^1.0.11",
     "npm-run-all": "^3.0.0",
     "nyc": "^8.3.0",
+    "object.map": "^0.2.0",
```

Comment:
We're working really hard to pare down our external dependencies. We have an unmaintainable number of them, contributing to very slow installs, complexity, and occasional upstream bugs. Would you mind rewriting this functionality in yourself in our `util` module or doing the object map using built-in primitives? I apologize -- I know this is a bummer.